### PR TITLE
move newapp commands that need docker into extended tests

### PR DESCRIPTION
**Pull request by bparees**

### CODE REVIEW

**Comment by bparees** [comment]
[test][extended]

**Comment by bparees** [comment]
@deads2k @smarterclayton let the cargo culting begin.

File: test/extended/test-cmd-docker.sh [hunk 1]
```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
```
Comment (smarterclayton) [inline test/extended/test-cmd-docker.sh]: test/extended/cmd.sh

Comment (bparees) [inline test/extended/test-cmd-docker.sh]: renamed

**Comment by bparees** [comment]
@smarterclayton anything else?

**Comment by smarterclayton** [comment]
LGTM [merge]

**Pull request merged**